meetings/agenda.md

```diff
@@ -7,3 +7,48 @@ Please file pull requests to add, or discuss items to add, to the agenda.
 
 ## Items to discuss
 
+### Components to be potentially moved to security-only status
+
+Proposal: these components should (probably) be clearly labeled so that maintainers know that only security
+issues are to be fixed, if they occur.
+
+ * laminas-db
+ * laminas-developer-tools
+ * laminas/laminas-oauth
+ * laminas/laminas-xml
+ * laminas/laminas-composer-autoloading
+ * laminas/laminas-auradi-config
+ * laminas/laminas-development-mode
+ * laminas/documentation
+ * laminas/laminas-barcode
+ * laminas/laminas-config-aggregator-parameters
+ * laminas/laminas-config
+ * laminas/laminas-config-aggregator-modulemanager
+ * laminas/laminas-console
+ * laminas/laminas-dom
+ * laminas/laminas-crypt
+ * laminas/laminas-mvc-console
+ * laminas/laminas-json
+ * laminas/laminas-http
+ * laminas/laminas-memory
+ * laminas/laminas-loader
+ * laminas/laminas-servicemanager-di
+ * laminas/laminas-json-server
+ * laminas/laminas-mime
+ * laminas/laminas-log
+ * laminas/laminas-math
+ * laminas/laminas-xml2json
+ * laminas/laminas-soap
+ * laminas/laminas-paginator
+ * laminas/laminas-navigation
+ * laminas/laminas-progressbar
+ * laminas/laminas-serializer
+ * laminas/laminas-tag
+ * laminas/laminas-text
+ * laminas/laminas-twitter
+ * laminas/laminas-uri
+ * laminas/laminas-mail
+ * laminas/laminas-file
+ * laminas/laminas-di
```

Comment:
This is actively developed, and the maintainer has been doing an excellent job of providing features. (In fact, he's even provided ways to integrate with laminas-servicemanager, and provides AoT compilation.) This is something I'd like to see as part of our tutorials, actually.